File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/25/25.java
// Q25: Print number pyramid (1, 232, 34543, 4567654).
// Input: An integer n
// Output: Number pyramid pattern

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        for (int i = 1; i <= n; i++) {
            for (int j = 0; j < n - i; j++) {
                System.out.print(" ");
            }
            for (int j = 0; j < i; j++) {
                System.out.print(i + j);
            }
            for (int j = i - 2; j >= 0; j--) {
                System.out.print(i + j);
            }
            System.out.println();
        }
    }
}

File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/25/25.js
// Q25: Print number pyramid (1, 232, 34543, 4567654).
// Input: An integer n
// Output: Number pyramid pattern

const readline = require('readline');
const rl = readline.createInterface({ input: process.stdin });
rl.on('line', (line) => {
    const n = parseInt(line);
    for (let i = 1; i <= n; i++) {
        let row = " ".repeat(n - i);
        for (let j = 0; j < i; j++) {
            row += (i + j);
        }
        for (let j = i - 2; j >= 0; j--) {
            row += (i + j);
        }
        console.log(row);
    }
    rl.close();
});
